inbox/models/category.py

```diff
@@ -18,6 +19,36 @@
 EPOCH = datetime.utcfromtimestamp(0)
 
 
+def sanitize_name(name):
+    # g_label may not have unicode type (in particular for a numeric
+    # label, e.g. '42'), so coerce to unicode.
+    name = unicode(name)
```

Comment:
I believe that doing `unicode(unicode(name))` fails, so we may need to wrap this in a check to make sure `name` is not already `isinstance(name, unicode)`